app/views/shared/check_answers/_final_hearing.html.erb

```diff
@@ -1,4 +1,3 @@
-<% read_only = false unless local_assigns.key?(:read_only) %>
```

Comment:
This partial is only ever called from `_proceeding_details`. At the top of that partial is the same line:
`<% read_only = false unless local_assigns.key?(:read_only) %>`
Because we're passing `read_only` into this partial, it's already doing this anyway, isn't it? I thought repeating the line was redundant. Let me know if you agree with my thinking